editor/src/components/canvas/controls/size-box.tsx

```diff
@@ -128,9 +128,7 @@ interface ResizeEdgeProps {
   dragState: DragState | null
 }
 
-interface ResizeEdgeState {
-  showLabel: boolean
-}
+interface ResizeEdgeState {}
```

Comment:
This should just be deleted now then

Reply:
I just kept it because it didn't even seem worth removing.

Reply:
Can you delete it please since there's no reason to keep it